projects/ngx-prx-styleguide/src/public_api.ts

```diff
@@ -1,29 +1,47 @@
 export { ChartsModule } from './lib/charts/charts.module';
```

Comment:
a change in the public_api: It now needs to include all components that are exported by the ng modules. I initially tried to use barrel files and reduce the amount of import statements, but that didn't work for the library to be consumed by applications. It's never a fun time to guess that it's barrel files that are causing you errors and headaches, so I also went around a couple places and totally got rid of em.